vtv-shows-app/src/typings/show.ts

```diff
@@ -0,0 +1,14 @@
+// src/typings/show.ts
+export interface IReview {
+    text: string;
+    rating: number;
+}
+  
+export interface IShow {
+    title: string;
+    description: string;
+    averageRating?: number;
+    imageUrl?: string;
+    reviews?: IReview[];
```

Comment:
The same thing, and this is OK, but in Infinum, we prefer the `Array<IReview>` syntax instead of `IReview[]`